HeterogeneousCore/CUDAUtilities/interface/HistoContainer.h

```diff
@@ -0,0 +1,161 @@
+#ifndef HeterogeneousCoreCUDAUtilitiesHistoContainer_h
+#define HeterogeneousCoreCUDAUtilitiesHistoContainer_h
+
+
+#include<cassert>
+#include<cstdint>
+#include<algorithm>
+#ifndef __NVCC__
+#include<atomic>
+#endif
+#ifdef __NVCC__
+#include <cuda_runtime.h>
+#endif
+
+#include "HeterogeneousCore/CUDAUtilities/interface/cudastdAlgorithm.h"
+
+#ifdef __NVCC__
+
```

Comment:
I would put all these symbols in a namespace, e.g. cudautils, to avoid collisions - especially for a common name like `zero`.

Reply:
ok